src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -867,7 +867,15 @@ private function registerSessionConfiguration(array $config, ContainerBuilder $c
         // session storage
         $container->setAlias('session.storage', $config['storage_id'])->setPrivate(true);
         $options = ['cache_limiter' => '0'];
-        foreach (['name', 'cookie_lifetime', 'cookie_path', 'cookie_domain', 'cookie_secure', 'cookie_httponly', 'use_cookies', 'gc_maxlifetime', 'gc_probability', 'gc_divisor', 'use_strict_mode'] as $key) {
+
+        $possibleOptions = ['name', 'cookie_lifetime', 'cookie_path', 'cookie_domain', 'cookie_secure', 'cookie_httponly', 'use_cookies', 'gc_maxlifetime', 'gc_probability', 'gc_divisor', 'use_strict_mode'];
+
+        if (\PHP_VERSION_ID > 70100) {
```

Comment:
this code is missing from branch 4.3, which means it doesn't work there neither

Reply:
Hmm. It looks like symfony has almost the same code for configuring session settings here https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php#L358 and here https://github.com/symfony/symfony/blob/4.3/src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php#L380. Am I missing something? Also I don't understand, why `NativeSessionStorage` has `sid_length` and `sid_bits_per_character` in both branches, but I cannot use it because it is not accessible via configuration?

Reply:
Sorry, I missed configuration part. Moving forward

Reply:
Fixed

Reply:
So, as explained, we're not going to merge this PR on 3.4, since it's a new feature.
But this very change is needed on 4.3. Let's rebase/retarget for 4.3 as a bug fix.
Thanks.

Reply:
I will create bugfix PR for 4.3. But at the same time I think that it is bugfix for 3.4 because it is the same as fix for https://github.com/symfony/symfony/issues/31089. It is changed api and should be supported because you don't have limit for supported php version here and this is maintained branch